modules/installation-vsphere-installer-infra-static-ip-nodes.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies for vSphere:
+//
+// * installing_vsphere/installing-vsphere-installer-provisioned.html
+// * installing_vsphere/installing-vsphere-installer-provisioned-customizations.html
+// * installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.html
+// * installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.html
+
+[comment]
+--
+* xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc[Installing a cluster on vSphere].
+--
+
+ifeval::["{context}" == "installing-vsphere-installer-provisioned"]
+:vsphere:
+endif::[]
+ifeval::["{context}" == "installing-vsphere-network-customizations"]
+:vsphere:
+endif::[]
+ifeval::["{context}" == "installing-vsphere-installer-provisioned-network-customizations.html"]
+:vsphere:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-installer-provisioned-vsphere.html"]
+:vsphere:
+endif::[]
+
+:_content-type: CONCEPT
+[discrete]
+[id="installation-vsphere-installer-infra-static-ip-nodes_{context}"]
+== Static IP addresses for vSphere nodes
+
+You can provision bootstrap, control plane, and compute nodes to use static IP addresses, so that you can better configure your network to meet the security demands of your organization. To configure this environment, you must provide values to the `platform.vsphere.hosts.role` parameter in the `install-config.yaml` file.
+
+:FeatureName: Static IP addresses for vSphere nodes
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+By default, the installation program is configured to use the Dynamic Host Configuration Protocol (DHCP) for the network, but this network has limited configurable capabilities.
+
+After you define `controlPlane`, `compute` or both machine pools in your `install-config.yaml` file, you can define network defintions for nodes on your network. Ensure that the number of network definitions match the configured machine pools types.
```

Comment:
SME query: I'm trying to avoid the use of the word master because of [conscious language initiatives](https://redhat-documentation.github.io/supplementary-style-guide/#conscious-language), so I hope this sentence is still clear?

Reply:
SME query: Only for control plane and compute nodes?

Ensure that the number of network definitions match the configured machine pools types.

Reply:
oh man, yeah, i try hard to avoid that work as well.  control plane and compute nodes is fine.